backend/api/views/dataset.py

```diff
@@ -84,4 +86,17 @@ def datawork_import(self, request):
         queryset = new_datasets.annotate(Count("files")).select_related("dataset_type")
         serializer = self.serializer_class(queryset, many=True)
 
+        try:
+            check_new_spectro_config()
+        except FileNotFoundError as error:
+            regex = "dataset/(.*)/analysis"
+            buggy_dataset = re.findall(regex, str(error))
```

Comment:
Please don't use regex unless it saves a lot of lines :wink:, use more `split` and simple string operations.

Reply:
Ok, I have a problem with regex. I want to put anywhere  :sweat: